connector/protobuf/src/test/scala/org/apache/spark/sql/protobuf/ProtobufCatalystDataConversionSuite.scala

```diff
@@ -138,6 +138,7 @@ class ProtobufCatalystDataConversionSuite
         data != null &&
         (data.get(0) == defaultValue ||
           (dt.fields(0).dataType == BinaryType &&
+            data.get(0) != null &&
```

Comment:
We should allow null value for the field. I.e. change this to:
   ```scala
     data.get(0) == null || data.get(0).asInstanceOf[Array[Byte]].isEmpty
  ```

I will test on my side to ensure this test stays stable with range of `seed` values.
cc: @HeartSaVioR

Reply:
Ignore this suggestion. `data.get(0) != null` is correct.  It allows testing null.